packages/ui-lib/src/Misc/atoms/Tag.tsx

```diff
@@ -47,43 +63,52 @@ export const TagWrapper = styled.div<{ hoverColor:ISvgIcons['color']; enableHove
   `};
 `;
 
+export type TypeTagSize = 'compact' | 'default'
```

Comment:
Well the third option that is not so obviously is "undefined".
The implemented logic cares for the scenario this  value is not sent to respect the previous versions where the size of the tag was the same as the size of text. 

Do you think it should go to legacy and only keep default and compact options?

Reply:
As we have the prop set with the `?` as `tagSize?: TypeTagSize`, I don't think we need the `undefined` here as it should be clear enough.

While I'd like to just switch it to a boolean as those are the only two size options planned, it seems that as it would be unnecessarily breaking so maybe we can circle round to it when get to a point where we can more actively break and refactor things.